astropy/modeling/utils.py

```diff
@@ -484,3 +484,64 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image, x_key='x_mean', y_key='y_mean'):
```

Comment:
Why are the `x_key` and `y_key` arguments necessary? Is it because the `Gaussian2D` and `Gaussian1D` have different parameter names then `x_0` and `y_0` (as the other models)? In that case, I'd just assume the parameters are called  `x_0` and `y_0` and handle an exception for the Gaussian case (In fact one should maybe think of renaming the Gaussian parameters, because `mean` and `stddev` are terms only used in statistics... but that would be a major discussion and I don't think want to deal with that...  )